R/usage.R

```diff
@@ -98,17 +103,10 @@ showMetrics <- function(metricSeries, metricNames, appDir = getwd(), appName = N
     stop("No data.", call. = FALSE)
   }
 
-  # get data points
-  points <- data$points
-  points <- lapply(points, function(X) {
-    X$time <- X$time / 1000 # convert from milliseconds to seconds
-    X
-  })
-
-  # convert to data frame
-  df <- data.frame(matrix(unlist(points), nrow = length(points), byrow = TRUE), stringsAsFactors = FALSE)
-  colnames(df) <- c(metricNames, "timestamp")
-  return(df)
+  points <- lapply(data$points, as.data.frame, stringsAsFactors = FALSE)
+  points <- do.call(rbind, points)
+  points$time <- .POSIXct(points$time / 1000)
```

Comment:
This was previously called `timestamp`, but that code has been failing for some time, and I think it's better to keep the column names as close as possible to what shinyapps.io returns?